ext/SparseMatrixColoringsColorsExt.jl

```diff
@@ -48,19 +35,19 @@ ncolors(res::AbstractColoringResult{s,:row}) where {s} = maximum(row_colors(res)
 
 function SparseMatrixColorings.show_colors(
     res::AbstractColoringResult;
-    colorscheme=DEFAULT_COLOR_SCHEME,
+    colorscheme=distinguishable_colors(
+        ncolors(res), [RGB(1, 1, 1), RGB(0, 0, 0)]; dropseed=true
+    ),
```

Comment:
Maybe break this out into a `default_colorscheme(n)` function and document what the arguments do?

Reply:
I looks like this excludes `RGB(1, 1, 1)` and `RGB(0, 0, 0)` (black and white). Maybe we can be smarter by only excluding `background`?

Reply:
<img width="580" alt="image" src="https://github.com/user-attachments/assets/66c76268-b42b-41f8-899d-f97970653f7e">

Reply:
yeah that's even better

Reply:
I meant your suggestion to just exclude the background is even better